hack/release.toml

```diff
@@ -25,6 +25,12 @@ runc: 1.1.11
 Flannel: 0.24.1
 
 Talos is built with Go 1.21.6.
+"""
+
+    [notes.device_selectors]
+        title = "Device Selectors"
+        description = """\
+Talos now supports `physical: true` qualifier for device selectors.
```

Comment:
probably needs to be more clear what physcial is, like physically attached NIC's

Reply:
yep, I will clarify this, thanks